geode-core/src/main/java/org/apache/geode/internal/DSFIDFactory.java

```diff
@@ -1058,30 +1059,22 @@ private static InterestResultPolicyImpl readInterestResultPolicy(DataInput in)
   private static DataSerializableFixedID readDestroyOnDataStore(DataInput in)
       throws IOException, ClassNotFoundException {
     DataSerializableFixedID serializable = new DestroyRegionOnDataStoreMessage();
-    serializable.fromData(in);
+    serializable.fromData(in, null);
```

Comment:
These (3) calls to `fromData()` with context `null` concern be because I wonder what the recipient is going to do when context is `null`.

I see maybe 200 implementations of `DataSerializableFixedID`. All of those outside membership are still accessing the static data serializer (not referring to the context at all).

Is there a way to get ahold of a context here and pass it along, if only to prevent problems when we eventually go into the 200-odd implementations of DSFID and sever the ties to the static data serializer in favor of the context parameter?

Reply:
oh, good catch - I meant to deal with that before opening a PR

Reply:
done